src/main/java/com/samsolutions/kitayeu/myproject/entities/Employee.java

```diff
@@ -0,0 +1,94 @@
+package com.samsolutions.kitayeu.myproject.entities;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+import java.util.Set;
+
+@Entity
+@Table(name="employee")
+public class Employee {
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    private int employeeId;
+    private String firstname;
+    private String lastname;
+    private LocalDate birthdate;
+    private char gender;
+    private String passportId;
+    private LocalDate passportValidity;
+    @ManyToMany (cascade = CascadeType.ALL)
+    @JoinTable (name = "employee_role",joinColumns = @JoinColumn (name ="employee_id"),
+    inverseJoinColumns = @JoinColumn(name="role_id"))
+    Set<Role> role;
```

Comment:
Why it is not private?

Reply:
I fixed and changed several items but can`t myselft resolve the gradle tests error